public/docs/ts/latest/guide/glossary.jade

```diff
@@ -0,0 +1,445 @@
+:markdown
+  # Angular 2 Glossary
+#sg-tables.showcase.shadow-1
+  header.showcase-header
+    p.
+      <i>The difference between the right word
+      and the almost right word is the difference between
+      lightening and a lightening bug.</i> - Mark Twain
+
+:markdown
+  Angular 2 has a vocabulary of its own.
+  Most Angular 2 terms are everyday English words 
+  with a specific meaning within the Angular system.
+  
+  We have gathered here the most prominent terms
+  and a few less familiar ones that have unusual or
+  unexpected definitions.
+
+.l-main-section
+:markdown
+  ## Annotation
```

Comment:
Can we use "Decorator" instead? Because that's simply what they are. They have been annotations when traceur was used...